### PR TITLE
[MINOR] Specify branch names for Github action

### PR DESCRIPTION
**Pull request by jongyoul**
### What is this PR for?
Specify the condition to trigger GitHub action when pushing commits. E.g. dependabot's PRs run twice for GitHub action as we allow to run it for all branches in the repository.

### What type of PR is it?
Bug Fix

### Todos
* [x] - Add `branches:` information for `push:`

### What is the Jira issue?
N/A

### How should this be tested?
* Check dependabot's PR so that it runs only once for each Github actions

### Questions:
* Does the licenses files need to update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: .github/workflows/core.yml [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
 on:
   push:
+    branches:
+      - master
+      - branch-*
   pull_request:
     branches:
       - master
       - branch-*
-    types: [opened, synchronize]
```
Comment (jongyoul) [inline .github/workflows/core.yml:8]: > By default, a workflow only runs when a pull_request event's activity type is opened, synchronize, or reopened.
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

**Review by pan3793 — DISMISSED** [review]

File: .github/workflows/core.yml [hunk 2]
```diff
@@ -1,11 +1,14 @@
 name: core
+
 on:
   push:
+    branches:
```
Comment (Reamer) [inline .github/workflows/core.yml:5]: The GitHub CI pipeline is turned off with this configuration in developer branches within forks.

Comment (jongyoul) [reply .github/workflows/core.yml:5]: I also thought so. Let me finding a solution so that it should be adopted in apache repository :-)

Comment (jongyoul) [reply .github/workflows/core.yml]: I changed it to use `branches-ignores` instead. :-)

**Review by Reamer — APPROVED** [review]
LGTM

**Pull request merged**